x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/TextExpansionQueryIT.java

```diff
@@ -103,8 +103,16 @@ public class TextExpansionQueryIT extends PyTorchModelRestTestCase {
         RAW_MODEL_SIZE = Base64.getDecoder().decode(BASE_64_ENCODED_MODEL).length;
     }
 
+    public void testRankFeaturesTextExpansionQuery() throws IOException {
```

Comment:
I've duped the tests on text_expansion queries so we check they can be used both with `rank_features` and `sparse_vector` field types.

I could as well have created separate classes for this, LMK if you can think of better ways to handle this.

Reply:
Lets not do this. Thank you for taking initiative here. But, all the ML related changes should be done after sparse_vector is merged. It will be good to separate the changes to make the change log cleaner and reviews easier.

Reply:
Got it - I'll revert this change and the `text_expansion_search_sparse_vector.yml` test as well.